### PR TITLE
Change classic readme in order to avoid missunderstanding

### LINKED ISSUE #21514: Classic theme readme is wrong
#### Describe the bug

On the classic readme, it's saying that the Classic theme is new, and based on the "new starter theme" which looks wrong now, it's not new and the starter theme is not maintained anymore.

#### Expected behavior

Not having "new theme, new starter" inside the readme

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | The readme of the classic theme could lead to missunderstanding because the starter theme is not maintained anymore, also it's not a "new" theme and the starter theme is not new too...
| Type?         | improvement
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Issue ? | Fixes #21514.
| How to test?  | No tests needed

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/21513)

### CODE REVIEW

**Review by PierreRambaud — DISMISSED** [review]

File: themes/classic/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # PrestaShop Classic Theme
 
-The Classic Theme is the new default Theme for Prestashop 1.7 and onwards.
-It is build upon the new Starter Theme.
+The Classic Theme is the default Theme for Prestashop 1.7 and onwards.
+It is build upon the Starter Theme wich is not maintained anymore.
```
Comment (SimonGrn) [inline themes/classic/README.md]:
```suggestion
It is built upon the Starter Theme wich is not maintained anymore.
```
I'm not sure ?

Comment (NeOMakinG) [reply themes/classic/README.md]: I think you're right

File: themes/classic/README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
 # PrestaShop Classic Theme
 
-The Classic Theme is the new default Theme for Prestashop 1.7 and onwards.
-It is build upon the new Starter Theme.
+The Classic Theme is the default Theme for Prestashop 1.7 and onwards.
+It is built upon the Starter Theme wich is not maintained anymore.
```
Comment (SimonGrn) [inline themes/classic/README.md]:
```suggestion
It is built upon the Starter Theme which is not maintained anymore.
```

Comment (SimonGrn) [reply themes/classic/README.md]: I'm pretty sure that's the last one :grin:

**Review by SimonGrn — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Pull request merged**